src/tests/rocketsProfile.test.js

```diff
@@ -0,0 +1,13 @@
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import RocketsProfile from '../components/rocketsProfile';
+import store from '../redux/configureStore';
+
+test('matches RocketsProfile component snapshot', () => {
+  const tree = renderer.create(
+    <Provider store={store}>
+      <RocketsProfile />
+    </Provider>,
+  ).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] In all tests you have only implemented snapshot tests. Snapshot tests only assert that the components are rendering correctly. The functionality of the app still needs to be tested. Please write some more tests that test the real functionalities of the app. For example:

1. Test that asserts that Rockets are loaded on the home page
2. Test that asserts that Missions are loaded on the missions page
3. Test that asserts that a user can click on the mission link and the app navigates to the missions page
4. Test that asserts that a user can join a mission
5. Test that asserts that a user can leave a mission
6. Test that asserts that a user can book a rocket
7. Test that asserts that a user can cancel rocket booking
8. Test that asserts that a user can click on the My Profile link and the app navigates to the My Profile page